### PR TITLE
use vagrant with install instructions

### PR DESCRIPTION
**Pull request by rogusdev**
Everything installs and works now :)  Takes about 5-10 minutes start to finish.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,71 @@
...
+    brew cask install virtualbox vagrant
+
+On Linux you should be able to figure this out yourself. ;)
+
+
+Download the database zip from the link you should have received and put it in this directory.
+
+Run Vagrant:
+
+    vagrant up && vagrant ssh
```
Comment (gtelljohann) [inline README.md]: my computer didn't have the bento ubuntu for vagrant, so I had to do `vagrant box add https://app.vagrantup.com/bento/boxes/ubuntu-18.04`

can you add that to the readme?

Comment (gtelljohann) [reply README.md]: actually I had old versions of vagrant and virtualbox that I didn't update, so maybe the newer version would have known to download it

Comment (rogusdev) [reply README.md]: vagrant definitely downloads missing boxes for you.  You can test by deleting the downloaded one and seeing if it downloads it automatically now that you have upgraded vagrant

`$HOME/.vagrant.d/boxes` mac
`%USERPROFILE%\.vagrant.d\boxes` windows

**Comment by gtelljohann** [comment]
Sounds good, I'll try again with the latest vagrant version

File: README.md (hunk 1 above)
Comment (gtelljohann) [reply README.md]: yup, you're right, it hadn't downloaded with the old version of vagrant, but when I updated it worked fine

**Comment by Astraeus-** [comment]
@Fabian-Sommer Tried a clean install with the windows instructions and it worked for me without any issues.

**Pull request merged**